spec/menu_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'menu'
+require 'order'
+
+describe Menu do
+# dependence injection. Inject dishes hash to menu.
+  subject(:menu) { described_class.new(dishes) }
+
+  let(:dishes) do
```

Comment:
This structure for dishes is neat.  An alternative...

```
[
{name: 'lasagne', price: 8.99}
]
```

Here, the keys are consistent and that is sometimes useful when you're iterating.